manifests/quickstart-feature-examples/job-level-affinity/podGroup.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: scheduling.godel.kubewharf.io/v1alpha1
+kind: PodGroup
+metadata:
+  generation: 1
+  name: nginx
+spec:
+  affinity:
+    podGroupAffinity:
+      preferred:
+        - topologyKey: netminiipv4
```

Comment:
can we rename this topology key ?

Reply:
I renamed the keys to be 'micronet' and 'mainnet', please see if this makes sense.